asaplib/data/xyz.py

```diff
@@ -90,7 +90,7 @@ def get_descriptors(self, desc_name=None, use_atomic_desc=False):
 
         return desc, atomic_desc
 
-    def get_property(self, y_key=None):
+    def get_property(self, y_key=None, extensive=True):
```

Comment:
shall we write some documentation for this, and set out some rules of what we want from the `asapXYZ`?